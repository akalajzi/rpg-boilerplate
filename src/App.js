// @flow strict
import React from "react"
import { BrowserRouter } from "react-router-dom";


const App = () => {
  return(
    <BrowserRouter>
      <div>asdf</div>
    </BrowserRouter>
  )
}

export default App
