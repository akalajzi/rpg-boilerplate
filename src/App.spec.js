// @flow strict
import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

// TODO: dont snapshot this, write meaningful test

describe('#App', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<App />);

    expect(wrapper.getElement()).toMatchSnapshot();
  });
});
