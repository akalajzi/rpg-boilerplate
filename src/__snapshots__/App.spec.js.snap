// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#App should render correctly 1`] = `
<ThemeProvider
  theme={
    Object {
      "colors": Object {
        "black": "#010101",
        "orange": "#FFA500",
        "white": "#FFF",
      },
    }
  }
>
  <Routes />
</ThemeProvider>
`;
